tests/base/utils/lsvCLI.ts

```diff
@@ -0,0 +1,164 @@
+import { spawn } from 'child_process';
+
+type RoleAssignment = {
+  account: string; // Ethereum address
+  role: `0x${string}`; // hex role identifier
+};
+
+type CreateVaultParams = {
+  defaultAdmin: string;
+  nodeOperator: string;
+  nodeOperatorManager: string;
+  confirmExpiry: number;
+  nodeOperatorFeeRate: number;
+  quantity?: number;
+  roles?: RoleAssignment[];
+  deployedFile?: string;
+};
+
+type VaultCreationResult = {
+  vaultAddress: string;
+  dashboardAddress: string;
+};
+
+export const createVault = async (
+  params: CreateVaultParams,
+): Promise<VaultCreationResult> => {
+  const {
+    defaultAdmin,
+    nodeOperator,
+    nodeOperatorManager,
+    confirmExpiry,
+    nodeOperatorFeeRate,
+    quantity = 1,
+    roles = [],
+  } = params;
+
+  return new Promise((resolve, reject) => {
+    const args = [
+      'contracts',
+      'factory',
+      'write',
+      'create-vault',
+      defaultAdmin,
+      nodeOperator,
+      nodeOperatorManager,
+      String(confirmExpiry),
+      String(nodeOperatorFeeRate),
+      String(quantity),
+    ];
+
+    if (roles.length > 0) {
+      args.push('--yes', '--roles', JSON.stringify(roles));
+    }
+
+    const cli = spawn('yarn', ['lsvCLI', ...args], {
+      stdio: ['pipe', 'pipe', 'pipe'],
+    });
+
+    let stdout = '';
+    let stderr = '';
+
+    cli.stdout.on('data', (data) => {
+      stdout += data.toString();
+    });
+
+    cli.stderr.on('data', (data) => {
```

Comment:
looks like the second wheel

Reply:
Fixed. Switched to passing `PRIVATE_KEY` through the subprocess environment via `yarn` instead of modifying the global `process.env`, avoiding side effects across tests.